multimodal/setup.py

```diff
@@ -25,15 +25,14 @@
     'Pillow',
     'tqdm',
     'boto3',
-    'setuptools<=59.5.0',
     'timm<0.6.0',
-    'torch>=1.0,<1.11',
+    'torch>=1.0,<1.12',
```

Comment:
You need to update this in Tabular: https://github.com/awslabs/autogluon/blob/master/tabular/setup.py#L42

Reply:
Let me try it out after our CI is fixed.

Reply:
I just tested autogluon successfully with torch 1.13 (nightly build). So I wonder, is there a need to be so conservative about torch versions, or can we just require 1.x? Have there be any cases of changes being necessary to support torch minor version upgrades?

Reply:
I just checked the version requirement of pytorch_lightning: https://github.com/Lightning-AI/lightning/blob/554056c21ae191b7bd2ea669d9ac84917dacdc6d/requirements/base.txt#L2 . They are also strict for torch and we may follow what they are doing.

Reply:
> I just checked the version requirement of pytorch_lightning: https://github.com/Lightning-AI/lightning/blob/554056c21ae191b7bd2ea669d9ac84917dacdc6d/requirements/base.txt#L2 . They are also strict for torch and we may follow what they are doing.

makes sense to me: torch does not guarantee strict semver2 as they sometimes ship breaking changes with a minor version release (also the case e.g. with 1.11)